### ISSUE TITLE
stddev computes wrong answer

### ISSUE THREAD

**Issue by johnnaegle** [brycebaril/node-stats-lite#8]
I don't know what is going on here, but when I use these particular numbers, the standard deviation is wrong..

```javascript
  t.deepEquals(stdev([710, 660.474666666667, 337.83, 600, 550, 710]), 140.6576424, "stdev works")
```

```
  ---
    operator: deepEqual
    expected: 140.6576424
    actual:   128.40227272732838
    at: Test.<anonymous> (/Users/johnnaegle/code/github/node-stats-lite/test/index.js:122:5)
  ...
```

Here is google sheets answer for the same numbers:

![screen shot 2018-08-31 at 8 19 20 pm](https://user-images.githubusercontent.com/1682292/44940961-348bb380-ad5b-11e8-81e2-7ffbca8a4d69.png)

**Comment by brycebaril** [comment]
This library calculates "Population Standard Deviation" and it appears that spreadsheets use "Sample Standard Deviation". We could probably add the second one as another function if you're interested!

![selection_114](https://user-images.githubusercontent.com/126482/44941207-e7551500-ad4d-11e8-89e9-066ccf42944b.png)
![selection_113](https://user-images.githubusercontent.com/126482/44941210-ecb25f80-ad4d-11e8-927b-a09bf429d7b1.png)

**Comment by johnnaegle** [comment]
I just got to this point as well, thanks for the quick answer.  It has been a "fun" refresher on statistics.  I'll probably make a PR for the stdev sample version.  Thanks

**Comment by johnnaegle** [comment]
Fixed by #9
